src/test/java/uk/gov/companieshouse/web/accounts/controller/smallfull/AddOrRemoveLoansControllerTest.java

```diff
@@ -141,6 +150,54 @@ void postDirectorAddRequestThrowsServiceException() throws Exception {
                 .andExpect(view().name(MOCK_CONTROLLER_PATH));
     }
 
+    @Test
+    @DisplayName("Post add loan - throws validation errors")
+    void postDirectorAddRequestThrowsValidationErrors() throws Exception {
```

Comment:
Is the name of this method correct? Should we refer to `Loan` rather than `Director`?

Reply:
This has just been picked up in the main PR